cmd/notary-server/config.json

```diff
@@ -13,6 +13,6 @@
 	},
 	"storage": {
 		"backend": "mysql",
-		"db_url": "dockercondemo:dockercondemo@tcp(notarymysql:3306)/dockercondemo"
+		"db_url": "gordontheturtle:dockerrulez@tcp(notarymysql:3306)/development"
```

Comment:
oooooo i see, but in the container i see this being a problem maybe...

On Fri, Jul 17, 2015 at 1:06 PM, Richard Scothern <notifications@github.com>
wrote:

> In cmd/notary-server/config.json
> <https://github.com/docker/notary/pull/76#discussion_r34926147>:
>
> > @@ -13,6 +13,6 @@
> >  	},
> >  	"storage": {
> >  		"backend": "mysql",
> > -		"db_url": "dockercondemo:dockercondemo@tcp(notarymysql:3306)/dockercondemo"
> > +		"db_url": "gordontheturtle:dockerrulez@tcp(notarymysql:3306)/development"
>
> I have no idea what Cobra is :)
>
> This thing will be deployed with Ansible, until infra switches to
> docker-compose, so changes to this file are for now immaterial
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/docker/notary/pull/76/files#r34926147>.
>